saleor/discount/migrations/0050_old_sale_ids_sequence.py

```diff
@@ -24,4 +24,21 @@ class Migration(migrations.Migration):
             ],
             reverse_sql=["DROP SEQUENCE discount_promotion_old_sale_id_seq"],
         ),
+        migrations.RunSQL(
+            sql=[
+                """
+            CREATE SEQUENCE discount_promotionrule_old_channel_listing_id_seq
+            OWNED BY discount_promotionrule.old_channel_listing_id;
+
+            SELECT setval(
+                'discount_promotionrule_old_channel_listing_id_seq',
+                 coalesce(max(old_channel_listing_id), 0) + 1000, false
```

Comment:
Will 1000 be safe? Do we know how much new `PromotionRule` can be created by users during migration?

Reply:
We'll need to rethink this value, as the migration from old sale to new promotions will happen in the background for the previous version, and for the start `old_channel_listing_id` will be empty for all promotions